chall01/ihwang.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import sys
+
+def intp_to_mos(av):
+	for i in range(len(av)):
```

Comment:
no need to iterate by index in Python in this case
you can do it like that:
`for c in av:`
and c will be each character of av (or each element if it's a list or dict)